controllers/storagecluster/cephcluster_test.go

```diff
@@ -102,6 +103,53 @@ func TestEnsureCephCluster(t *testing.T) {
 	}
 }
 
+func TestCephClusterMonTimeout(t *testing.T) {
+	// cases for testing
+	cases := []struct {
+		label    string
+		platform v1.PlatformType
+	}{
+		{
+			label:    "case 1",
+			platform: v1.NonePlatformType,
+		},
+		{
+			label:    "case 2",
+			platform: v1.IBMCloudPlatformType,
+		},
+		{
+			label:    "case 3",
+			platform: IBMCloudCosPlatformType,
+		},
+	}
+
+	for _, c := range cases {
+		sc := &api.StorageCluster{}
+		mockStorageCluster.DeepCopyInto(sc)
+		sc.Status.Images.Ceph = &api.ComponentImageStatus{}
+
+		reconciler := createFakeStorageClusterReconciler(t, mockCephCluster)
+
+		reconciler.platform = &Platform{
+			platform: c.platform,
+		}
+
+		var obj ocsCephCluster
+		err := obj.ensureCreated(&reconciler, sc)
+		assert.NoError(t, err)
+
+		cc := newCephCluster(sc, "", 3, reconciler.serverVersion, nil, log)
+		err = reconciler.Client.Get(context.TODO(), mockCephClusterNamespacedName, cc)
+		assert.NoError(t, err)
+		if c.platform == v1.IBMCloudPlatformType || c.platform == IBMCloudCosPlatformType {
+			fmt.Println(c.platform)
```

Comment:
is the print statement needed here? if yes, then we can add some more context to it. 
say, 
`fmt.Println("Platform:", c.platform)`

Reply:
was using this to debug and forgot to remove.
Its not required